pkg/clustermesh/kvstoremesh/kvstoremesh_test.go

```diff
@@ -279,6 +283,18 @@ func TestRemoteClusterRun(t *testing.T) {
 				}, timeout, tick, "Expected key %q does not seem to have the correct value %q", key, value)
 			}
 
+			expectedNotReflected := map[string]string{
+				"cilium/cache/identities/v1/foo/value/bar": "qux5",
+			}
+			// Assert that the keys have not been reflected
+			for key, value := range expectedNotReflected {
+				require.EventuallyWithTf(t, func(c *assert.CollectT) {
+					v, err := kvstore.Client().Get(ctx, key)
+					assert.NoError(c, err)
+					assert.NotEqual(c, value, string(v))
+				}, timeout, tick, "Key %q is not expected to be synced, got a match on value %q", key, value)
+			}
```

Comment:
Let's switch this test to:
```
			// Assert that other keys have not been reflected
			values, err := kvstore.Client().ListPrefix(ctx, "cilium/cache/identities/v1/")
			require.Len(t, values, 1)
```
Checking if something doesn't exists with "eventually" probably doesn't make sense here. 
Instead, we can just check if anything else was not propagated in that specific prefix, including "value" keys.

Reply:
Thanks for the suggestion, updated in 53b6fc2a9d